temporalio/worker/_workflow_instance.py

```diff
@@ -69,6 +69,36 @@
     WorkflowOutboundInterceptor,
 )
 
+from temporalio.service import __version__
+
+@dataclass
+class SDKInfo:
+    name    : str
+    version : str
+        
+@dataclass
+class FileSlice:
+    content    : str
+    lineOffset : int
+
+@dataclass
+class FileLocation:
+    filePath     : str
+    line         : Optional[int] = -1
+    column       : Optional[int] = -1
+    functionName : Optional[str] = None
+        
+@dataclass
+class StackTrace:
+    locations : list[FileLocation]
+        
+@dataclass
+class EnhancedStackTrace:
+    sdk     : SDKInfo
+    sources : dict[str, FileSlice]
+    stacks  : list[StackTrace]
```

Comment:
These might belong in the `common` module rather than here. In either case they probably don't need to be readily visible to users, so I'd probably prefix them with `_`. If they stay here, I'd move them further down the file with other classes like this.

Also make sure you run the autoformatter (`poe format`)

Reply:
Refactored them to `temporalio.common`!